apps/desktop/src/lib/gitHost/gitlab/gitlab.ts

```diff
@@ -1,13 +1,14 @@
-import { GitLabBranch } from './gitlabBranch';
 import type { RepoInfo } from '$lib/url/gitUrl';
 import type { GitHost } from '../interface/gitHost';
 import type { DetailedPullRequest, GitHostArguments } from '../interface/types';
+import { GitLabBranch } from './gitlabBranch';
```

Comment:
Well this now assumes that folks are putting their Gitlab instance at `gitlab.company.com` always, right?

What do you think about makign this a user setting? @mtsgrd